dts/arm/nuvoton/npcx/npcx4.dtsi

```diff
@@ -14,6 +14,8 @@
 #include "npcx4/npcx4-espi-vws-map.dtsi"
 /* npcx4 series low-voltage io controls mapping table */
 #include "npcx4/npcx4-lvol-ctrl-map.dtsi"
+/* npcx4 series reset mapping table*/
```

Comment:
`/* npcx4 series reset mapping table */`

(missing space before the comment closing marker)

Reply:
Done